src/components/TodosList/TodosList.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[],
+  isAdding: boolean,
+  title: string,
+  removeTodo: (value: number) => void,
+  setSelectedTodos: (value: number[]) => void,
+  selectedTodos: number[],
+  onUpdate: (todoId: number, data: Partial<Todo>) => void,
+};
+
+export const TodosList: React.FC<Props> = ({
+  todos,
+  isAdding,
+  title,
+  removeTodo,
+  setSelectedTodos,
+  selectedTodos,
+  onUpdate,
+}) => {
+  const temp = {
+    id: 0,
+    title,
+  };
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      <TransitionGroup>
+        {todos.map(todo => (
+          <CSSTransition
+            key={todo.id}
+            timeout={300}
+            classNames="item"
+          >
+            <TodoItem
+              todo={todo}
+              key={todo.id}
```

Comment:
Why did you add `key` here and the same in line 35?

Reply:
🤷‍♂️